src/test/java/com/synopsys/integration/detect/testutils/ContainerScanTestUtils.java

```diff
@@ -51,11 +55,23 @@ public OperationRunner setUpDetectConfig(BlackduckScanMode blackduckScanMode, St
         DetectConfigurationFactory detectConfigurationFactory = makeContainerScanFactory(blackduckScanMode, imageFilePath);
         BootSingletons bootSingletonsMock = Mockito.mock(BootSingletons.class);
         Mockito.when(bootSingletonsMock.getDetectConfigurationFactory()).thenReturn(detectConfigurationFactory);
+
+        // Create mocked operation wrapper
```

Comment:
nit: looks like you're mocking `UtilitySingletons`, not `OperationWrapper`.

Reply:
Refactored, thanks